src/notification/email/EMailNotificationTask.ts

```diff
@@ -141,87 +144,126 @@ export default class EMailNotificationTask implements NotificationTask {
+      this.backupInUse = useBackup;
+    }
     // Create the message
-    const messageToSend = {
-      from: (!retry ? this.emailConfig.smtp.from : this.emailConfig.smtpBackup.from),
+    const messageToSend = new Message({
+      from: !this.emailConfig.disableBackup && !Utils.isUndefined(this.emailConfig.smtpBackup) && useBackup ? this.emailConfig.smtpBackup.from : this.emailConfig.smtp.from,
       to: email.to,
       cc: email.cc,
-      bcc: (email.bccNeeded ? email.bcc : ''),
+      bcc: email.bccNeeded && email.bcc ? email.bcc : '',
       subject: email.subject,
-      // pragma text: email.text
       attachment: [
         { data: email.html, alternative: true }
       ]
-    };
-    // Send the message and get a callback with an error or details of the message that was sent
-    return this[!retry ? 'server' : 'serverBackup'].send(messageToSend, async (err, messageSent) => {
-      if (err) {
-        // If authentication error in the primary email server then notify admins using the backup server
-        if (!retry && this.serverBackup) {
-          // TODO: Circular deps: src/notification/NotificationHandler.ts -> src/notification/email/EMailNotificationTask.ts -> src/notification/NotificationHandler.ts
-          NotificationHandler.sendSmtpAuthError(
-            tenant.id,
-            {
-              'evseDashboardURL': data.evseDashboardURL
-            }
-          ).catch(() => {});
-        }
-        // Log
-        try {
-          Logging.logError({
-            tenantID: tenant.id,
-            source: (Utils.objectHasProperty(data, 'chargeBoxID') ? data.chargeBoxID : undefined),
-            action: ServerAction.EMAIL_NOTIFICATION,
-            module: MODULE_NAME, method: 'sendEmail',
-            message: `Error Sending Email (${messageToSend.from}): '${messageToSend.subject}'`,
-            actionOnUser: user,
-            detailedMessages: [
-              {
-                email: {
-                  from: messageToSend.from,
-                  to: messageToSend.to,
-                  subject: messageToSend.subject
-                },
-              }, {
-                error: err.stack
-              }, {
-                content: email.html
-              }
-            ]
-          });
-          // For Unit Tests only: Tenant is deleted and email is not known thus this Logging statement is always failing with an invalid Tenant
-        } catch (error) {}
-        // Retry?
-        if (!retry && this.serverBackup) {
-          return this.sendEmail(email, data, tenant, user, severity, true);
-        }
-      } else {
-        // Email sent successfully
-        Logging.logDebug({
-          tenantID: tenant.id,
-          source: (Utils.objectHasProperty(data, 'chargeBoxID') ? data.chargeBoxID : undefined),
+    });
+    try {
+      // Send the message
+      const messageSent: Message = await this.getSMTPClient().sendAsync(messageToSend);
```

Comment:
Simply use useBackup var to get the right SMTP client.
Something like this:
`return useBackup ? SMTPBackupClientInstance : SMTPMainClientInstance`
No need this.backupInUse member var